### ISSUE TITLE
Android animation flickering

### ISSUE THREAD

**Issue by wvengen** [jeanregisser/react-native-popover#7]
This component works on Android too, but before the animation starts, the end frame is shown briefly, causing flickering. (Using React 0.18.)

To see it more clearly, this is a recording with animations slowed down 10 times.
![popover-android-slow](https://cloud.githubusercontent.com/assets/503804/12414455/c3b2ad6e-be95-11e5-87ae-437988b6d2e5.gif)

**Comment by roselind** [comment]
The  Arrow is  disappeared  on android ,  
 
Why is that so?
 
How do you modify this bug?

**Comment by lprhodes** [comment]
@wvengen did you ever find a solution for this?

**Comment by wvengen** [comment]
@lprhodes nope, we moved to [Picker](http://facebook.github.io/react-native/docs/picker.html#content) (see also [this](https://github.com/facebook/react-native/issues/3004#issuecomment-202598006), if that's still needed).
